app/models/replication/obj_proxy.rb

```diff
@@ -156,34 +162,36 @@ def get_copy_ancestry(context)
     else
       # We combine the copy parent's own ancestry (which may be nil) plus its ID.
       joined = [context[:copy_parent].ancestry, context[:copy_parent].id].compact.join("/")
-      joined.blank? ? nil : joined
+      joined.presence
     end
   end
 
-  def date_col_mappings(replicator, context)
+  def date_col_mappings(_replicator, _context)
     [["created_at", "NOW()"], ["updated_at", "NOW()"]]
   end
 
-  def unique_col_mappings(replicator, context)
+  def unique_col_mappings(replicator, _context)
     return [] unless uniq_spec = klass.replicable_opts[:uniqueness]
+
     generator = Replication::UniqueFieldGenerator.new(
-      uniq_spec.merge(klass: klass, orig_id: id, mission_id: replicator.target_mission_id))
+      uniq_spec.merge(klass: klass, orig_id: id, mission_id: replicator.target_mission_id)
+    )
     [[uniq_spec[:field], ApplicationRecord.connection.quote(generator.generate)]]
   end
 
-  def standardizable_col_mappings(replicator, context)
+  def standardizable_col_mappings(replicator, _context)
     mappings = []
     case replicator.mode
     when :promote
       mappings << ["is_standard", true] if klass.standardizable?
     when :clone
-      mappings << "mission_id"
+      mappings << "mission_id" if klass_has_mission_id
       mappings << "is_standard" if klass.standardizable?
     when :to_mission
-      mappings << ["mission_id", quote_or_null(replicator.target_mission_id)]
+      mappings << ["mission_id", quote_or_null(replicator.target_mission_id)] if klass_has_mission_id
```

Comment:
Using `klass_has_mission_id` here is needed for `Tagging` which does not have mission id.